specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/preview/2023-10-01-preview/cognitiveservices.json

```diff
@@ -5631,25 +5631,16 @@
       "description": "Azure OpenAI Content Filter.",
       "type": "object",
       "properties": {
-        "policyName": {
+        "name": {
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

Why change it from policyName to name? PolicyName seems clear, and its a breaking change, is this reviewed?

Reply:
Thanks for the review. This parameter is not a policy name, it's a **filter name**. This breaking change is more like a bug fix, we've got the breaking change approval while the review meeting.